src/components/DocExplorer.js

```diff
@@ -78,7 +83,15 @@ export class DocExplorer extends React.Component {
   }
 
   _onSearch = event => {
-    this.setState({ searchValue: event.target.value });
+    if (event.target.value === '') {
```

Comment:
Is there any way this event could be triggered twice where both trigger this condition? If so then the nav will go backwards twice. For example, are there keys that can be pressed such that when the keyup event below is fired, we accidentally interpret it as a "nav back" event?